lib/winston-azure-application-insights.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module "winston-azure-application-insights" {
+  import type * as appInsights from "applicationinsights";
+  import type * as Transport from "winston-transport";
```

Comment:
Should just be `import Transport from "winston-transport";`

Reply:
seems to be ok TS both ways, but on module it's just "export = " so changed to use directly.

Reply:
It yelled at me when trying to import the module into another typescript file. Did you not have that problem?